src/platform_impl/linux/event_loop.rs

```diff
@@ -640,7 +640,7 @@ impl<T: 'static> EventLoop<T> {
                   }
                 }
               }
-              glib::Propagation::Proceed
+              glib::Propagation::Stop
```

Comment:
How does this change affects Widgets added inside the window, do they still receive events? or will be swallowed by these handlers here?

Reply:
Is there a way I can easily test that?

Reply:
you can get the gtk window created by tao using https://docs.rs/tao/latest/tao/platform/unix/trait.WindowExtUnix.html#tymethod.gtk_window and then you can create a gtk Button https://docs.rs/gtk/latest/gtk/struct.Button.html and add it to the window using `gtk_window.add()`, then you will need to add these same handlers on the button you created, and ensure:
1. the button can be clicked normally
2. the handlers on the button are triggered

Reply:
The button worked fine, I checked all three event handlers and they fire correctly as well. Should be good to go!